unstructured/ingest/connector/s3_connector.py

```diff
@@ -175,7 +175,8 @@ def initialize(self):
             raise ValueError(
                 f"No objects found in {self.config.s3_url} -- response list object is {response}",
             )
-        os.mkdir(self.config.download_dir)
+        if not Path(self.config.download_dir).exists():
+            os.mkdir(self.config.download_dir)
```

Comment:
This can be abbreviated to one line: `os.mkdir(self.config.download_dir, exist_ok=True)`.

It feels kind of odd that this would only need to be added to this connector, though.

Reply:
OK, looking more closely at s3_connector, this dir does not need to be created here in the first place because any time a file is downloaded, all parent dirs are created if needed:
https://github.com/Unstructured-IO/unstructured/blob/e52dd5c/unstructured/ingest/connector/s3_connector.py#L86 .
Tested to confirm, but can go ahead and remove this block.

Reply:
Ok I'll update both that suggestion and the changelog too, thanks 👍🏻